superset-frontend/cypress-base/cypress/integration/dashboard/filter.test.ts

```diff
@@ -30,7 +30,7 @@ describe('Dashboard filter', () => {
     cy.visit(WORLD_HEALTH_DASHBOARD);
   });
 
-  it('should apply filter', () => {
+  xit('should apply filter', () => {
```

Comment:
I've tested this manually, and can't reproduce the reason for this test to fail. Disabling for now as Filter Box is about to be deprecated soon (will take another stab at enabling this later once I get my local cypress env up and running).

Reply:
Turns out there was a timeout issue on cypress - fixed in the helper util